### PR TITLE
Fix/first time launch issue

### PR DESCRIPTION
**Pull request by rleojoseph**
# Description
Mini App fails to launch for the very first time due to the async process in Signature verification. Downloaded tmp files deleted automatically before receiving the response from Signature SDK.
Now we are making to return the Signature verification status once the downloading is complete. 

## Links
[MINI-4550](https://jira.rakuten-it.com/jira/browse/MINI-4550)

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: Sources/Classes/core/Downloader/MiniAppClient.swift [hunk 1]
```diff
@@ -264,10 +267,23 @@ internal class MiniAppClient: NSObject, URLSessionDownloadDelegate {
...
+        let tempPathDirName = (ids?.0 ?? "") + "/" + (ids?.1 ?? "")
+        guard let tempLocation = self.delegate?.moveFileToTempLocation(from: location, to: tempPathDirName) else {
+            delegate?.downloadFileTaskCompleted(url: "", error: NSError.downloadingFailed())
+            return
+        }
+        checkFileSignature(destinationURL: destinationURL, location: tempLocation)
     }
 
     private func checkFileSignature(destinationURL: String, location: URL) {
+        MiniAppLogger.d("LEO: checkFileSignature", destinationURL)
```
Comment (Climbatize) [inline Sources/Classes/core/Downloader/MiniAppClient.swift:286]: do we still need this log? If yes I suggest something without `LEO` inside it ^^

File: Sources/Classes/core/Downloader/MiniAppClient.swift [hunk 2]
```diff
@@ -264,10 +267,23 @@ internal class MiniAppClient: NSObject, URLSessionDownloadDelegate {
             delegate?.downloadFileTaskCompleted(url: "", error: NSError.downloadingFailed())
             return
         }
-        checkFileSignature(destinationURL: destinationURL, location: location)
+        fileDownloadingCompleted(destinationURL: destinationURL, at: location)
+    }
+
+    private func fileDownloadingCompleted(destinationURL: String, at location: URL) {
```
Comment (Climbatize) [inline Sources/Classes/core/Downloader/MiniAppClient.swift:273]: maybe `fileDownloadingCompleted` should be renamed `fileDownloadCompleted`, but I'm not sure. Also I think we got the `destinationURL` wrong from the beginning, it should be `originURLString`, isn't it?

**Review by Climbatize — APPROVED** [review]
👏 👏 👏

File: Sources/Classes/core/Downloader/MiniAppClient.swift (hunk 2 above)
Comment (rleojoseph) [reply Sources/Classes/core/Downloader/MiniAppClient.swift:273]: Let me check

Comment (rleojoseph) [reply Sources/Classes/core/Downloader/MiniAppClient.swift]: That's right. It's a bit confusing. Renamed it now.

**Review by laubengaier — APPROVED** [review]
lgtm nice!

**Pull request merged**